### PR TITLE
Add jsonLimit setting that reads from ENV var

### PR DESCRIPTION
**Pull request by ibarreto**
This PR adds a `jsonLimit` setting so we can set the size for `express.json()`.

### CODE REVIEW

File: src/robot.js [hunk 1]
```diff
@@ -441,7 +442,7 @@ class Robot {
     }
     app.use(express.query())
 
-    app.use(express.json())
+    app.use(express.json({ jsonLimit }))
```
Comment (ilyoff) [inline src/robot.js]: according to documentation, there is no jsonLimit parameter, but limit

Comment (ilyoff) [inline src/robot.js]:
```suggestion
    app.use(express.json({ limit: jsonLimit }))
```

**Comment by ibarreto** [comment]
@ilyoff good catch, thanks

**Comment by ibarreto** [comment]
This has been a very painful PR process, closing

**Pull request closed without merging**